### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by gshilin**

### CODE REVIEW

**Comment by amnonbb** [comment]
Maybe better put language selection inside settings menu?

**Comment by alexmbb** [comment]
For now put it outside so everyone see, later one we reorganize it

**Comment by gshilin-sdb** [comment]
It is outside

File: src/translations.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+export const resources = {
+  en: {
+    common: {
+      openChat: 'Open Chat',
```
Comment (edoshor) [inline src/translations.js]: This flat list of keys is not really scaleable. I would love to see these grouped and nested into a reasonable hierarchy.

File: src/App.js [hunk 2]
```diff
@@ -16,31 +19,58 @@ import OldClient from "./apps/VirtualApp/OldClient";
 // import SDIOutApp from "./apps/SDIOutApp/SDIOutApp";
 // import AdminCongress from "./apps/AdminApp/AdminCongress";
 // import AdminStreaming from "./apps/AdminApp/AdminStreaming";
+import { resources } from './translations';
 
-class App extends Component {
+// instance for client side
+i18n
```
Comment (edoshor) [inline src/App.js]: Can we extract this init into its own file ? 
Let's keep `App.js` as small as possible

**Pull request merged**